src/pages/[id].page.jsx

```diff
@@ -1,4 +1,7 @@
 import { useState, useEffect } from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+// eslint-disable-next-line no-unused-vars
+import { amethyst, cyberpunk, levelUp } from '@codesandbox/sandpack-themes';
```

Comment:
Love these names 😄
I think we can delete the ones we're not using for now, though. We can revisit in a later PR if we want to.

Reply:
The theme names were great! Just looking at names, I was a little disappointed NightOwl and Dracula didn't work. 

Anyways, they're deleted! https://github.com/sparkbox/accessible-components/pull/105/commits/1fa508dd12b7bc45b33ba214d1f04fd1ea5dae89